deploy/osd-cluster-ready/60-osd-ready.Job.yaml

```diff
@@ -11,10 +11,15 @@ spec:
     template:
         metadata:
             name: osd-cluster-ready
+            annotations:
+                # Keep this in sync with image
+                managed.openshift.io/imageTag: v0.1.45-c9f1f45
```

Comment:
Let's make this a label, as that seems to be what's done elsewhere.

(Aside: that doesn't make a lot of sense to me, as I can't think of a scenario where you would want/need to be able to search by a version.)

Reply:
Done.